File: Java/Flip-Bits.java
class Solution {
    /**
     *@param a, b: Two integer
     *return: An integer
     */
    public static int bitSwapRequired(int a, int b) {
        // write your code here
        int cnt = a ^ b, res = 0;
        while (cnt != 0) {
            cnt = cnt & (cnt - 1);
            res ++;
        }
        return res;
    }
};

File: Python/Flip-Bits.py
class Solution:
    """
    @param a, b: Two integer
    return: An integer
    """
    def bitSwapRequired(self, a, b):
        # write your code here
        cur = a ^ b
        res = 0
        while cur != 0:
            res += 1
            cur = (cur & (cur - 1))
        return res
